cmd/build

```diff
@@ -2,7 +2,7 @@
 
 if [ "$1" == help ]; then
  echo The following are expected modes of builds:
- echo $0 : defaults to build-all
+ echo $0 : defaults to build 
```

Comment:
but I think it needs to default to build-all?  Maybe the default is wrong... But until the dev workflow is out we have to have correctness over speed.

Reply:
the current default is build based on the code so I updated the help doc to reflect that. it has no impact on integration testing if that's what you're concerned about.